src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -381,25 +381,25 @@ void func_80AEB220(EnRu1* this, PlayState* play) {
 }
 #endif
 
-void func_80AEB264(EnRu1* this, AnimationHeader* animation, u8 arg2, f32 morphFrames, s32 arg4) {
+void EnRu1_AnimationChange(EnRu1* this, AnimationHeader* animation, u8 mode, f32 morphFrames, s32 direction) {
     s32 pad[2];
     AnimationHeader* animHeader = SEGMENTED_TO_VIRTUAL(animation);
```

Comment:
direction could be `playReversed` and passed in as `true/false`. You would change the if in this function to be `if (!playReversed) {`